### ISSUE TITLE
No unconfigured devices found

### ISSUE THREAD

**Issue by constantindjonkam** [PlusPlus-ua/ha_tuya_ble#43]
When I add this integration, it says, "No unconfigured devices found."

**Comment by PlusPlus-ua** [comment]
It just means you have no Tuya BLE devices in range of your Bluetooth adapter. If you have some Tuya Bluetooth devices, they might be Bluetooth Mesh, not BLE.

**Comment by NikolayTLM** [comment]
> It just means you have no Tuya BLE devices in range of your Bluetooth adapter. If you have some Tuya Bluetooth devices, they might be Bluetooth Mesh, not BLE.

same here! may be my CUBETOUCH II (Device ID: bfffe16xpvyzqcdd) is not known by your integration?

**Comment by niho9807** [comment]
Same here with the Tuya TH05 Temperature and Humidity sensor. I placed the sensor like 10cm away from my RasPi and it won't detect it.

**Comment by yalov** [comment]
Does it support all T&H sensors `category_id: wsdcg`, or just the one `product_id: ojzlzzsw` from that category? 
I have `"model": "TH05", "category": "wsdcg", "product_id": "vyfoip9h", "product_name": "Temp & Humidity Sensor"`

Does adding Bluetooth device to default Tuya integration 
https://www.home-assistant.io/integrations/tuya/ conflicts with this plugin ?
The device works on the Smart Life (Tuya Life) app, it is added to the default Tuya Home Assistant integration, but does not show values.

I have tried to use this, but "No unconfigured devices found"
Also I run HomeAssistant OS in the virtualbox on a laptop for now, maybe it is related

UPD. no values on the default integration: https://github.com/tuya/tuya-home-assistant/issues/932

**Comment by SJM1001** [comment]
Having the same issue, I got the fingerbot added, Kept on saying offline. I then decided to remove the integration to retry the process. Now it says No unconfigured devices found, but everything is in exactly the same positions (within Bluetooth range) . Is there a way to maybe see if it stored data of the previous time? Then if I can remove that data I can perhaps try getting the device configured again.

**Comment by gitothy** [comment]
> Does it support all T&H sensors `category_id: wsdcg`, or just the one `product_id: ojzlzzsw` from that category? I have `"model": "TH05", "category": "wsdcg", "product_id": "vyfoip9h", "product_name": "Temp & Humidity Sensor"`
> 
> Does adding Bluetooth device to default Tuya integration https://www.home-assistant.io/integrations/tuya/ conflicts with this plugin ? The device works on the Smart Life (Tuya Life) app, it is added to the default Tuya Home Assistant integration, but does not show values.
> 
> I have tried to use this, but "No unconfigured devices found" Also I run HomeAssistant OS in the virtualbox on a laptop for now, maybe it is related
> 
> UPD. no values on the default integration: [tuya/tuya-home-assistant#932](https://github.com/tuya/tuya-home-assistant/issues/932)

Yes exactly the same here. I also tried registering a completely separate Tuya account with a different device, account and IOT account. Paired the TH05 to this account and when adding to HA it says no devices found.  TH05 permanently shows as 'Offline' in Tuya IOT cloud even when the Device Debug screen shows data being periodically received.

**Comment by niho9807** [comment]
> Does it support all T&H sensors `category_id: wsdcg`, or just the one `product_id: ojzlzzsw` from that category? I have `"model": "TH05", "category": "wsdcg", "product_id": "vyfoip9h", "product_name": "Temp & Humidity Sensor"`
> 
> Does adding Bluetooth device to default Tuya integration https://www.home-assistant.io/integrations/tuya/ conflicts with this plugin ? The device works on the Smart Life (Tuya Life) app, it is added to the default Tuya Home Assistant integration, but does not show values.
> 
> I have tried to use this, but "No unconfigured devices found" Also I run HomeAssistant OS in the virtualbox on a laptop for now, maybe it is related
> 
> UPD. no values on the default integration: [tuya/tuya-home-assistant#932](https://github.com/tuya/tuya-home-assistant/issues/932)

Yesterday I've learned that the T&H sensors with product_id: vyfoip9h are Bluetooth Mesh devices and therefore not (yet) supported in @PlusPlus-ua 's work. But I honestly don't know if anything outside the Tuya BLE category will ever be supported in his work.

**Comment by gitothy** [comment]
OK thanks @niho9807 for the info.  I assume that being Bluetooth Mesh shouldn't preclude it from working with Home Assistant locally like any other BLE device (e.g. the Xiaomi LYWSD03MMC).

I guess the second hope is that the [ATC_MiThermometer](https://github.com/pvvx/ATC_MiThermometer) project expands to include the TH05 and similar Tuya BLE devices so we can flash and forget.

**Comment by JPM-git** [comment]
any news on this subject?

I have a CUBETOUCH II (Product Category: szjqr), which is supported, i get tuya integration to show the devices, I put the device on pair mode but nothing shows up: "No unconfigured devices found."

**Comment by jeroenvdd** [comment]
I seem to have same issue with a yiihr7zh,
but my whole house is full of ble devices and sensors etc close to the device seem to report just fine,
any way we can force this?

**Comment by MauricioXavier13** [comment]
> any news on this subject?
> 
> I have a CUBETOUCH II (Product Category: szjqr), which is supported, i get tuya integration to show the devices, I put the device on pair mode but nothing shows up: "No unconfigured devices found."

Did you manage to solve this issue?

**Comment by alemangioni** [comment]
Same with Product ID: ulughw4g (curtain motor). Can't detect it from HA and can't control it from Cloud since it's a local Bluetooth device with no IP. Can be only controlled by the Phone with any Smart App.

**Comment by evancodergit** [comment]
also cant detect bulb rgbcct 8002 4
